package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "5.16.6",
+  "version": "5.16.7",
```

Comment:
Potentially also update line 21 (shared approval type) when your other PR is merged.